arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift

```diff
@@ -23,32 +23,40 @@ class GraphicsOverlayDictionaryRenderer3DViewController: UIViewController {
     /// The scene view managed by the view controller.
     @IBOutlet weak var sceneView: AGSSceneView! {
         didSet {
-            sceneView.scene = AGSScene(basemapType: .imagery)
+            sceneView.scene = AGSScene(basemapStyle: .arcGISTopographic)
             sceneView.graphicsOverlays.add(makeGraphicsOverlay())
         }
     }
     
+    /// Joint Military Symbology MIL-STD-2525D.
+    let militaryItem = AGSPortalItem(
```

Comment:
Does this need to be a property? If so, I'm curious why. If not, please move it into `makeGraphicsOverlay()`.

Reply:
Thanks, no need to be a property.